src/applications/representative-search/config.js

```diff
@@ -67,7 +68,7 @@ export const resolveParamsWithUrl = ({
   page,
   perPage = 10,
   sort,
-  type = 'VSO',
+  type = 'veteran_service_officer',
```

Comment:
Maybe grab the type string from the `RepresentativeType` constant?

Reply:
Will include in next PR.